Doc/library/threading.rst

```diff
@@ -996,9 +996,8 @@ method.  The :meth:`~Event.wait` method blocks until the flag is true.
       (or fractions thereof).
 
       This method returns ``True`` if and only if the internal flag has been set to
-      true, either before the wait call or after the wait starts, so it will
-      always return ``True`` except if a timeout is given and the operation
-      times out.
+      true, either before the wait call or after the wait starts.
+      If a timeout is given and the operation times out, ``False`` will be returned.
```

Comment:
I don't understand what is the expected return value if no timeout is given and set() was not called: the function just doesn't return and block until set() is called?

Reply:
Several paragraphs above: 
```
      Block until the internal flag is true.  If the internal flag is true on
      entry, return immediately.  Otherwise, block until another thread calls
      :meth:`.set` to set the flag to true, or until the optional timeout occurs.
```

Reply:
My rewrite combines paragraphs is a way that I think is clearer in less words.  I think this ready to merge now.